tests/python-pytest/onnx/test_operators.py

```diff
@@ -338,17 +338,18 @@ def test_onnx_export_cast(tmp_path, src_dtype, dst_dtype, shape):
 
 
 @pytest.mark.parametrize('dtype', ['float16', 'float32'])
-def test_onnx_export_softmax(tmp_path, dtype):
+@pytest.mark.parametrize('temperature', [.1, 1., 10.])
```

Comment:
Should we check when temperature is not provided (and should default to 1.0?)

Reply:
Removed temperature from M1 to test the default case.